source/documentation/packaging.html.md

```diff
@@ -27,25 +28,10 @@ to begin opening up the technical governance of RDO.
 This process will be discussed and fleshed out in more detail on
 [rdo-list@redhat.com](https://www.redhat.com/mailman/listinfo/rdo-list).
 
-## Documentation
-
-### Packaging Guides
+## Packaging guides
 
 * [RDO Packaging Guide](../rdo-packaging)
 * [RDO Packaging Guidelines](../rdo-packaging-guidelines)
 * [`rdopkg` manual](/packaging/rdopkg/rdopkg.1.html)
 
-
-## Discussion Forums
```

Comment:
@hguemar I dropped it because I thought it somewhat duplicates content in https://www.rdoproject.org/community/